src/dev/ci_setup/setup.sh

```diff
@@ -30,13 +30,26 @@ fi
 ###
 ### download node
 ###
+UNAME=`uname`
+OS='linux'
+if [[ "$UNAME" = *"MINGW64_NT"* ]]; then
+  OS='win'
+fi
+echo "Running on OS: $OS"
```

Comment:
Mind throwing ` -- ` before this log message so it matches the other messages after `"Setting up node.js and yarn in $dir"`?